experiments/AMIP/modular/README.md

```diff
@@ -84,34 +84,34 @@ https://climate.ucdavis.edu/pubs/UMJS2013QJRMS.pdf
 # Heat Slab
 The slab solves for temperature in a single layer, whose tendency is the accumulated fluxes divided by the coupling timestep plus a parameterisation of the internal processes, $G$.
 $$
-\rho c h_s  \, \partial_t T_{sfc} =  - F_{integ}  / \Delta t_{coupler}  
+\rho c h_s  \, \partial_t T_{sfc} =  - F_{integ}  / \Delta t_{coupler}
 $$
 
 # Distribution
 ## Julia multithreading
-- we can run using multiple threads if the command line argument `enable_threading` is true (e.g., run with `julia --project --threads 8`). 
+- we can run using multiple threads if the command line argument `--device` is set to `CPUMultiThreaded` and the number of threads `n` is specified with `--threads n`.
```

Comment:
```suggestion
- we can run using multiple threads by specifying the number of threads `n` when launching Julia (e.g., `julia --threads n --project`).
```
I don't know that this is the right place for this documentation-- `--device` is a ClimaAtmos kwarg. I might recommend just deleting this doc string